jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/JsonExceptionMapper.java

```diff
@@ -36,15 +36,15 @@
  * Writes out exceptions as serialized {@link SerializableError}s with {@link MediaType#APPLICATION_JSON JSON media
  * type}.
  */
-abstract class JsonExceptionMapper<T extends Exception> implements ExceptionMapper<T> {
+public abstract class JsonExceptionMapper<T extends Exception> implements ExceptionMapper<T> {
 
-    private static final Logger log = LoggerFactory.getLogger(JsonExceptionMapper.class);
+    protected static final ObjectMapper MAPPER = getObjectMapper();
```

Comment:
if we expose this class outside the library, then the exception mapper must be made private. 
also, consider using `ObjectMappers.guavaJdk7().enable(SerializationFeature.INDENT_OUTPUT) in this class and in RemoteExceptionMapper.